File: benchmark/2026-09-17-7677581(latest)/standard/src/adversarial/zig/ADV-25_R.zig
const std = @import("std");

pub fn main() !void {
    var t = std.Io.Threaded.init_single_threaded;
    const io = t.io();
    const out = std.Io.File.stdout();
    try out.writeStreamingAll(io, "ADV-START\n");

    var ib: [64]u8 = undefined;
    var rd = std.Io.File.stdin().readerStreaming(io, &ib);
    const text: []const u8 = try rd.interface.takeSentinel('\n');

    const n: i64 = try std.fmt.parseInt(i64, text, 10);
    const r: i64 = n * 2;

    var ob: [64]u8 = undefined;
    try out.writeStreamingAll(io, try std.fmt.bufPrint(&ob, "OBS=R:{d}\n", .{r}));
    try out.writeStreamingAll(io, "ADV-END\n");
}

File: benchmark/2026-09-17-7677581(latest)/standard/src/adversarial/rust/ADV-25_R.rs
use std::io::BufRead;
use std::io::Write;

fn main() {
    let mut out = std::io::stdout();
    writeln!(out, "ADV-START").unwrap();
    out.flush().unwrap();

    let mut line = String::new();
    std::io::stdin().lock().read_line(&mut line).unwrap();
    let n: i64 = line.trim().parse::<i64>().unwrap();

    let r: i64 = n * 2;

    writeln!(out, "OBS=R:{}", r).unwrap();
    out.flush().unwrap();
    writeln!(out, "ADV-END").unwrap();
    out.flush().unwrap();
}
